howto/spark/spark_mlrun_read_csv.ipynb

```diff
@@ -0,0 +1,866 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Spark Job with MLRun\n",
+    "Basic read csv spark mlrun function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# nuclio: ignore\n",
+    "import nuclio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Function Kind and Base Image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get build base image name by running the following code\n",
+    "# copy and paste the output of this cell into config spec.build.baseImage\n",
+    "# this will use us to config our spark job docker image\n",
+    "import os\n",
+    "os.environ[\"IGZ_DATANODE_REGISTRY_URL\"] + '/iguazio/shell:' + os.environ[\"IGZ_VERSION\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "%nuclio: setting kind to 'job'\n",
+      "%nuclio: setting spec.build.baseImage to 'datanode-registry.iguazio-platform.app.hsbctesting3.iguazio-cd0.com:80/iguazio/shell:3.0_katyak_debug_b1089_20201214154653'\n"
+     ]
+    }
+   ],
+   "source": [
+    "# set the function kind and docker image\n",
+    "%nuclio config kind = \"job\"\n",
+    "%nuclio config spec.build.baseImage = \"datanode-registry.iguazio-platform.app.hsbctesting3.iguazio-cd0.com:80/iguazio/shell:3.0_katyak_debug_b1089_20201214154653\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Build MLRun Function"
```

Comment:
Please use another term then `build` since this is used when actually building the images on MLRun.